### PR TITLE
refactor(createRoutes): leverage default exports

### PR DESCRIPTION
**Pull request by devopsmariocom**
to simplify module loading logic

### CODE REVIEW

File: src/routes.tsx [hunk 1]
```diff
@@ -12,12 +12,8 @@ const loadModule = ( cb ) => ( componentModule ) => {
 };
```
Comment (Hotell) [inline src/routes.tsx]: cb needs annotation:

```
cb: (err:any, component: Component<any,any>)=>void
```

https://github.com/reactjs/react-router/blob/master/docs/API.md#getcomponentnextstate-callback

or check it in type definitions if there isn't a type for that already

Comment (Hotell) [inline src/routes.tsx]: hmm I don't see a point here why loadModule is a closure, when it can be loaded via one function call, see docs

**Comment by Hotell** [comment]
pls read this before doing anything https://github.com/reactjs/react-router/blob/master/docs/guides/DynamicRouting.md

**Comment by Hotell** [comment]
I'm not very comfortable to export things both by default and as named, but maybe for this particular reasons it makes sense.

Also the thing is that connected components should be exported by default and named for "testing" purposes

**Comment by devopsmariocom** [comment]
read https://github.com/reactjs/react-router/blob/master/docs/guides/DynamicRouting.md anything specific you want to point out?

**Comment by devopsmariocom** [comment]
ok so let's go with default exports for pages then same as for linked components

**Pull request merged**